packages/core/useResize/index.ts

```diff
@@ -198,16 +198,16 @@ export function useResize(element: MaybeElementRef, options: UseResizeOptions =
 
     const { left, top } = target.value!.getBoundingClientRect()
 
-    widthRef.value = clamp(newWidth, unref(minWidth as number), unref(maxWidth as number))
-    heightRef.value = clamp(newHeight, unref(minHeight as number), unref(maxHeight as number))
+    widthRef.value = clamp(newWidth, unref(Number(minWidth)), unref(Number(maxWidth)))
```

Comment:
I believe you will want `Number` outside the `unref` like `Number(unref(minWidth))`. Otherwise anytime `minWidth` is a  ref it will return NaN